gecode/plan.sh

```diff
@@ -0,0 +1,26 @@
+pkg_name=gecode
+pkg_distname=$pkg_name
+pkg_origin=core
+pkg_version=3.7.3
+pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
+pkg_license=('gecode')
+pkg_source=http://www.gecode.org/download/gecode-${pkg_version}.tar.gz
+pkg_shasum=e7cc8bcc18b49195fef0544061bdd2e484a1240923e4e85fa39e8d6bb492854c
+pkg_dirname=${pkg_distname}-${pkg_version}
+pkg_deps=(core/glibc core/gcc)
+pkg_build_deps=(core/make core/gcc core/coreutils core/diffutils core/perl)
+
```

Comment:
You'll want the following lines as well so that other packages can link against it and so that you get the couple of binaries gecode produces:
```
pkg_lib_dirs=(lib)
pkg_include_dirs=(include)
pkg_bin_dirs=(bin)
```